exec/nvbench-ctl.cu

```diff
@@ -0,0 +1,48 @@
+/*
+*  Copyright 2021 NVIDIA Corporation
+*
+*  Licensed under the Apache License, Version 2.0 with the LLVM exception
+*  (the "License"); you may not use this file except in compliance with
+*  the License.
+*
+*  You may obtain a copy of the License at
+*
+*      http://llvm.org/foundation/relicensing/LICENSE.txt
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*/
+
+#include <nvbench/main.cuh>
+
+#include <vector>
+
+int main(int argc, char const *const *argv)
+try
```

Comment:
TIL https://en.cppreference.com/w/cpp/language/function-try-block 👍

Reply:
@brycelelbach showed me that a while ago -- it's handy :)